Zeebe 0.22.1 debug-http exporter does not respect custom port

**Describe the bug**
Debug-http exporter is configured with a custom port but the port settings is ignored and it uses default 8000 port.

**To Reproduce**
1) Run an application on 8000 port so it is busy.
2) Configure the exporter to use other port, e.g. 8100 that is free.
3) Run the zeebe. It fails with (note **{port=8100.0, limit=1024.0}}**:
```
2020-04-16 12:12:26.384 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 partitions [1/1]: partition 1 failed with unexpected exception.
java.lang.IllegalStateException: Failed to load exporter with configuration: ExporterCfg{id='debug-http', jarPath='null', className='io.zeebe.broker.exporter.debug.DebugHttpExporter', args={port=8100.0, limit=1024.0}}
	at io.zeebe.broker.system.partitions.ZeebePartition.<init>(ZeebePartition.java:114) ~[zeebe-
```
**Expected behavior**

The port 8100 is used as configured. Note that if 8000 becomes free, it starts normally with:
```
2020-04-16 12:35:18.198 [] [main] INFO  io.zeebe.broker.exporter - Debug http server started, inspect the last 1024 records on http://localhost:8000
```
Regardless 8100 port is configured

```
[[exporters]]
id = "debug-http"
className = "io.zeebe.broker.exporter.debug.DebugHttpExporter"
[exporters.args]
port = 8100
limit = 1024
```

**Log/Stacktrace**
<details><summary>Full Stacktrace</summary>
 <p>

```
java.lang.IllegalStateException: Failed to load exporter with configuration: ExporterCfg{id='debug-http', jarPath='null', className='io.zeebe.broker.exporter.debug.DebugHttpExporter', args={port=8100.0, limit=1024.0}}
	at io.zeebe.broker.system.partitions.ZeebePartition.<init>(ZeebePartition.java:114) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.Broker.lambda$partitionsStep$15(Broker.java:316) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.bootstrap.StartProcess.lambda$startStepByStep$2(StartProcess.java:60) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.bootstrap.StartProcess.takeDuration(StartProcess.java:88) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.bootstrap.StartProcess.startStepByStep(StartProcess.java:58) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.bootstrap.StartProcess.takeDuration(StartProcess.java:88) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.bootstrap.StartProcess.start(StartProcess.java:43) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.Broker.partitionsStep(Broker.java:321) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.Broker.lambda$initStart$9(Broker.java:182) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.bootstrap.StartProcess.lambda$startStepByStep$2(StartProcess.java:60) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.bootstrap.StartProcess.takeDuration(StartProcess.java:88) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.bootstrap.StartProcess.startStepByStep(StartProcess.java:58) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.bootstrap.StartProcess.takeDuration(StartProcess.java:88) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.bootstrap.StartProcess.start(StartProcess.java:43) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.Broker.internalStart(Broker.java:130) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.util.LogUtil.doWithMDC(LogUtil.java:21) [zeebe-util-0.22.2.jar:0.22.2]
	at io.zeebe.broker.Broker.start(Broker.java:120) [zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.StandaloneBroker.main(StandaloneBroker.java:32) [zeebe-distribution-0.22.2.jar:0.22.2]
Caused by: io.zeebe.broker.exporter.repo.ExporterLoadException: Cannot load exporter [debug-http]: failed validation
	at io.zeebe.broker.exporter.repo.ExporterRepository.validate(ExporterRepository.java:94) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.repo.ExporterRepository.load(ExporterRepository.java:54) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.repo.ExporterRepository.load(ExporterRepository.java:85) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.system.partitions.ZeebePartition.<init>(ZeebePartition.java:112) ~[zeebe-broker-0.22.2.jar:0.22.2]
	... 17 more
Caused by: java.lang.RuntimeException: Failed to start debug exporter http server
	at io.zeebe.broker.exporter.debug.DebugHttpServer.startHttpServer(DebugHttpServer.java:76) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.debug.DebugHttpServer.<init>(DebugHttpServer.java:57) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.debug.DebugHttpExporter.initHttpServer(DebugHttpExporter.java:47) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.debug.DebugHttpExporter.configure(DebugHttpExporter.java:25) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.repo.ExporterRepository.validate(ExporterRepository.java:92) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.repo.ExporterRepository.load(ExporterRepository.java:54) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.repo.ExporterRepository.load(ExporterRepository.java:85) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.system.partitions.ZeebePartition.<init>(ZeebePartition.java:112) ~[zeebe-broker-0.22.2.jar:0.22.2]
	... 17 more
Caused by: java.net.BindException: Address already in use: bind
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:479) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:468) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:220) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:88) ~[?:?]
	at sun.net.httpserver.ServerImpl.<init>(ServerImpl.java:101) ~[jdk.httpserver:?]
	at sun.net.httpserver.HttpServerImpl.<init>(HttpServerImpl.java:50) ~[jdk.httpserver:?]
	at sun.net.httpserver.DefaultHttpServerProvider.createHttpServer(DefaultHttpServerProvider.java:35) ~[jdk.httpserver:?]
	at com.sun.net.httpserver.HttpServer.create(HttpServer.java:137) ~[jdk.httpserver:?]
	at io.zeebe.broker.exporter.debug.DebugHttpServer.startHttpServer(DebugHttpServer.java:71) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.debug.DebugHttpServer.<init>(DebugHttpServer.java:57) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.debug.DebugHttpExporter.initHttpServer(DebugHttpExporter.java:47) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.debug.DebugHttpExporter.configure(DebugHttpExporter.java:25) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.repo.ExporterRepository.validate(ExporterRepository.java:92) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.repo.ExporterRepository.load(ExporterRepository.java:54) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.exporter.repo.ExporterRepository.load(ExporterRepository.java:85) ~[zeebe-broker-0.22.2.jar:0.22.2]
	at io.zeebe.broker.system.partitions.ZeebePartition.<init>(ZeebePartition.java:112) ~[zeebe-broker-0.22.2.jar:0.22.2]
	... 17 more

```

</p>
</details>


**Environment:**
 - OS: Windows
 - Zeebe Version: 0.22.1
 - Configuration: 
```
[[exporters]]
id = "debug-http"
className = "io.zeebe.broker.exporter.debug.DebugHttpExporter"
[exporters.args]
port = 8100
limit = 1024
```

--- a/broker/src/main/java/io/zeebe/broker/exporter/debug/DebugHttpExporter.java
+++ b/broker/src/main/java/io/zeebe/broker/exporter/debug/DebugHttpExporter.java
@@ -44,11 +44,11 @@ public final class DebugHttpExporter implements Exporter {
       final DebugHttpExporterConfiguration configuration =
           context.getConfiguration().instantiate(DebugHttpExporterConfiguration.class);
 
-      httpServer = new DebugHttpServer(configuration.port, configuration.limit);
+      httpServer = new DebugHttpServer(configuration.getPort(), configuration.getLimit());
       log.info(
           "Debug http server started, inspect the last {} records on http://localhost:{}",
-          configuration.limit,
-          configuration.port);
+          configuration.getLimit(),
+          configuration.getPort());
     }
   }
 
@@ -66,7 +66,24 @@ public final class DebugHttpExporter implements Exporter {
   }
 
   public static class DebugHttpExporterConfiguration {
-    public final int port = 8000;
-    public final int limit = 1024;
+
+    private int port = 8000;
+    private int limit = 1024;
+
+    private int getPort() {
+      return port;
+    }
+
+    private void setPort(final int port) {
+      this.port = port;
+    }
+
+    private int getLimit() {
+      return limit;
+    }
+
+    private void setLimit(final int limit) {
+      this.limit = limit;
+    }
   }
 }

